### DISCUSSION TITLE
Smarters IPTV

### DISCUSSION THREAD

**Discussion by infxmousrogue** [iptv-org/community#582] [Q&A]
I am using smarters IPTV it keeps asking me for a required username and password. How to get it to work?

**Comment by HNJAMeindersma** [comment]
I'm assuming you've entered one of iptv-org `.m3u` file URL's in your IPTV app/software, like `https://iptv-org.github.io/iptv/languages/eng.m3u`? This URL is not password protected and does not ask for HTTP-Auth. So this seems to be a specific issue of the app/software that you are using.

You could try to send some fake Auth by changing the URL like this: `https://user:pass@iptv-org.github.io/iptv/languages/eng.m3u`. Please try this first.

Does the app/software that you use have a username and password field, besides the URL field when you setup a IPTV source?

**Reply by infxmousrogue** [reply]
@HNJAMeindersma Yes so the smarters pro app had a username and password field which are required to fill in.

I tried to keep them empty but it requires it either way. Tried with empty spaces and user test pass test but unfortunately the app does not seem compatible with this while the app is listed as a supported app tho.

**Reply by HNJAMeindersma** [reply]
Looks like the app was removed from the Android Play store. Maybe for a reason. Maybe it just want users to paste in working user credentials... Just a thought.

**Reply by ls-milkyway** [reply]
Nope, smarters iptv now requires to register your free personal m3u link by paying an amount ...they have also tied up with certain IPTV providers....and you can use only those provider's m3u links.

**Comment by Blackkumba** [comment]
@infxmousrogue did you resolve this problem ?

**Comment by akhanalcs** [comment]
Same issue for me.

**Comment by Amo0on101** [comment]
are you using the lite version or the pro one ?
i think the lite version has the option to add playlist

**Comment by Saharprince22** [comment]
French
